docs/core/logging.md

```diff
@@ -466,7 +466,7 @@ via `samplingRate` attribute on annotation.
 
 ## Upgrade to JsonTemplateLayout from deprecated LambdaJsonLayout configuration in log4j2.xml
 
-Prior to version [1.10.0](https://github.com/awslabs/aws-lambda-powertools-java/releases/tag/v1.10.0), only supported way of configuring `log4j2.xml` was via  `<LambdaJsonLayout/>`. This plugin is 
+Prior to version [1.10.1](https://github.com/awslabs/aws-lambda-powertools-java/releases/tag/v1.10.1), only supported way of configuring `log4j2.xml` was via  `<LambdaJsonLayout/>`. This plugin is 
```

Comment:
I think this is wrong

Reply:
yup ignore the pr for now.. i am trying to autmate this process